cvat-sdk/cvat_sdk/pytorch/__init__.py

```diff
@@ -70,6 +71,12 @@ class Target:
     """
 
 
+def _get_server_dir(client: cvat_sdk.core.Client) -> Path:
+    # Base64-encode the name to avoid FS-unsafe characters (like slashes)
+    server_dir_name = base64.urlsafe_b64encode(client.api_map.host.encode()).rstrip(b"=").decode()
+    return _CACHE_DIR / f"servers/{server_dir_name}"
```

Comment:
Could you extract the `_CACHE_DIR` variable into the `core.client.Config` class? The default value can be the same.

Reply:
Sure, I can do that. Note that this will mean that `appdirs` will become a core dependency (but I think that was going to happen sooner or later anyway).

Reply:
OK. Right, it's just a question of time.

Reply:
Implemented in #5535 (it seems like a separate feature, so I wanted to submit it separately).